terraform/ecs.tf

```diff
@@ -55,52 +64,54 @@ resource "aws_ecs_service" "webapp" {
 }
 
 resource "aws_ecs_task_definition" "service" {
-  family                   = "beacons-service-task"
+  family                   = "${var.env}-beacons-service-task"
   execution_role_arn       = aws_iam_role.ecs_task_execution_role.arn
   network_mode             = "awsvpc"
   requires_compatibilities = ["FARGATE"]
   cpu                      = var.service_fargate_cpu
   memory                   = var.service_fargate_memory
   container_definitions = jsonencode([{
-    "name" : "beacons-service",
-    "image" : "${var.service_image}:${var.service_image_tag}",
-    "portMappings" : [
+    name : "beacons-service",
+    image : "${data.aws_ecr_repository.service.repository_url}:${var.service_image_tag}",
+    portMappings : [
       {
-        "containerPort" : var.service_port
-        "hostPort" : var.service_port
+        containerPort : var.service_port
+        hostPort : var.service_port
       }
     ],
-    "environment" : [
+    environment : [
       {
-        "name" : "SPRING_DATASOURCE_URL",
-        "value" : "jdbc:postgresql://${aws_db_instance.postgres.endpoint}/${var.db_name}?sslmode=require"
+        name : "SPRING_DATASOURCE_URL",
+        value : "jdbc:postgresql://${aws_db_instance.postgres.endpoint}/${var.db_name}?sslmode=require"
       },
       {
-        "name" : "SPRING_DATASOURCE_USER",
-        "value" : var.db_username
-      },
-      {
-        "name" : "SPRING_DATASOURCE_PASSWORD",
-        "value" : var.db_password
+        name : "SPRING_DATASOURCE_USER",
+        value : var.db_username
       }
     ],
-    "logConfiguration" : {
+    logConfiguration : {
       "logDriver" : "awslogs",
       "options" : {
         "awslogs-group" : aws_cloudwatch_log_group.log_group.name,
         "awslogs-region" : var.aws_region,
         "awslogs-stream-prefix" : "service"
       }
-    }
+    },
+    secrets : [
+      {
+        name : "SPRING_DATASOURCE_PASSWORD",
+        valueFrom : aws_secretsmanager_secret.db_password.arn
```

Comment:
Passing this to ECS as a secret so it doesn't show up in plaintext in the task definition. #14